edge/pkg/metamanager/process.go

```diff
@@ -95,17 +95,6 @@ func msgDebugInfo(message *model.Message) string {
 	return fmt.Sprintf("msgID[%s] resource[%s]", message.GetID(), message.GetResource())
 }
 
-func resourceUnchanged(resType string, resKey string, content []byte) bool {
-	if resType == model.ResourceTypePodStatus {
-		dbRecord, err := dao.QueryMeta("key", resKey)
-		if err == nil && len(*dbRecord) > 0 && string(content) == (*dbRecord)[0] {
```

Comment:
for now, the pod status is updated when we detect changes between cache and apiserver, so the upstream podstatus message is infrequently，when we receive the update message, we must send it to the cloud.

Reply:
the pr contains 3 bug fixes? the title description misses the first commit info

Reply:
done